api/scripts/add_queue_message.py

```diff
@@ -0,0 +1,69 @@
+import os
```

Comment:
you need to make this file name something more relevant. like `schedule_processing_for_tokens_from_old_transfers.py`

Reply:
pretty much all scripts add messages to queues which is why this is not a great name